File: web-development/level-1/helsinki-part-1-2/Python/part1/week04/ex73.py/Product.py
class Product:
    #constructor
    def __init__(self, inital_name, initial_price, initial_amount):
        self.name = inital_name
        self.price = initial_price
        self.amount = initial_amount

    def print_product(self):
        return print('name: ', self.name,' price: ',self.price, 'amount: ',self.amount)

File: programming/helsinki-moocs/java/ex73/Product.java
public class Product {
  //private vars
  private String name;
  private int amount;
  private int price;

  //constructor
  public Product(String initial_name, int inital_amount, int initial_price ) {
    this.name = initial_name;
    this.amount = inital_amount;
    this.price = initial_price;
  }

  //methods
  public void print_product() {
    System.out.println(
        "Name: "+this.name+" | Amount: "+this.amount+" | Price: "+this.price+"$"
      );
  }
}
